openedx/core/djangoapps/discussions/models.py

```diff
@@ -2,19 +2,124 @@
 Provide django models to back the discussions app
 """
 from __future__ import annotations
+import logging
 
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
+from django_mysql.models import ListCharField
 from jsonfield import JSONField
 from model_utils.models import TimeStampedModel
 from opaque_keys.edx.django.models import LearningContextKeyField
 from simple_history.models import HistoricalRecords
 
 from lti_consumer.models import LtiConfiguration
+from organizations.models import Organization
 
+from openedx.core.djangoapps.config_model_utils.models import StackedConfigurationModel
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 
+log = logging.getLogger(__name__)
+
+
+def get_default_providers():
```

Comment:
@xitij2000 Is there currently a PR with plugin entrypoints that I can use to reference here?
I know there was previously, just unclear of its present status.

Alternatively, we could leave these blank for now (until the entrypoints merge) and/or we could even set defaults directly in the database (thanks to `StackedConfig`) .

Reply:
Since we're not using plugins at this point and directly using LTI I don't know how useful the old code will be. 

You can see it here: https://github.com/edx/edx-platform/pull/24584/files#diff-13929f35238ff0d4c17881898e899d6159ef1aaaadc9736c2df8fab05c2724efR1